widgets/serializers/react_fields.py

```diff
@@ -74,12 +74,25 @@ def _get_props(self):
 class ReactURLField(serializers.URLField, ReactField):
     """ReactField extension of DRF UrlField"""
 
+    def __init__(self, **kwargs):
+        if "show_embed" in kwargs:
+            self.show_embed = kwargs.pop("show_embed")
+        else:
+            self.show_embed = False
+
+        super().__init__(**kwargs)
+
+    def _get_input_type(self):
+        """Returns the field's input type"""
+        return "url"
+
     def _get_props(self):
         """Returns the field properties"""
         return {
             **super()._get_props(),
             "max_length": self.max_length or "",
             "min_length": self.min_length or "",
+            "show_embed": self.show_embed,
```

Comment:
what is this value used for?

Reply:
The RSS widget also accepts a URL input but I want to communicate to the frontend not to render it with embedly haha

Reply:
ok cool, was just wondering because I didn't see it used anywhere

Reply:
:thinking: If it's not in the frontend code anywhere then it's probably something I missed actually...

Reply:
Yeah I missed it. I just pushed a commit to check the flag.